P5/Source/Specs/noteGrp.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
+    Copyright TEI Consortium.
+    Dual-licensed under CC-by and BSD2 licences
+    See the file COPYING.txt for details
+    $Date$
+    $Id$
+--><?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?><elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="core" ident="noteGrp">
+  <desc versionDate="2005-01-14" xml:lang="en">contains a group of notes</desc>
+  <classes>
+    <memberOf key="att.global"/>
+    <memberOf key="model.noteLike"/>
+    <memberOf key="att.placement"/>
+    <memberOf key="att.pointing"/>
+    <memberOf key="att.typed"/>
+    <memberOf key="att.written"/>
+    <memberOf key="model.correspDescPart"/>
+    <memberOf key="model.correspActionPart"/>
+    <memberOf key="model.correspContextPart"/>
+  </classes>
+  <content>
+    <sequence>
+        <classRef key="model.headLike" minOccurs="0"/>
+        <elementRef key="desc" minOccurs="0" maxOccurs="unbounded"/>
+        <alternate minOccurs="1" maxOccurs="unbounded">
+          <elementRef key="note"/>
+          <elementRef key="noteGrp"/>
+        </alternate>
+    </sequence>
+  </content>
+  <attList>
+    <attDef ident="anchored" usage="opt">
```

Comment:
Rather than repeating all these remarks, should we perhaps move `@anchored` to its own class? Same goes for `@targetEnd`, I suppose.

Reply:
Not sure what's the procedure. Never done it myself.

Reply:
[Outline of procedure sent to @tuurma via Slack.]